docs/reference/system/schema.md

```diff
@@ -359,7 +359,7 @@ n/a
 
 ## Apply Schema Difference
 
-Update the instance's schema by passing the diff previously retrieved via `/schema/diff` endpoint in the request body.
+Update the instance's schema by passing the diff previously retrieved via `/schema/diff` endpoint in the request body - either as raw JSON or as a `multipart/form-data` file upload.
```

Comment:
I'd suggest
```suggestion
Update the instance's schema by passing the diff previously retrieved via `/schema/diff` endpoint in the request body.
```
and update `### Request Body` instead, like we have it in the other endpoints, see https://github.com/directus/directus/blob/33cd918213e4608833f15d68fab1ece32889f4e7/docs/reference/system/schema.md#request-body.